llama/sampling_ext.h

```diff
@@ -1,6 +1,6 @@
 // TODO: this is a temporary wrapper to allow calling C++ code from CGo
-#ifndef LLAMA_SAMPLING_EXT_H
-#define LLAMA_SAMPLING_EXT_H
+#ifndef gpt_sampler_EXT_H
+#define gpt_sampler_EXT_H
```

Comment:
Minor but it's usually more canonical to make this all caps (and the same for the comment at the bottom of the file).